src/FEValues/common_values.jl

```diff
@@ -78,8 +92,18 @@ quadrature point `q_point`.
 @propagate_inbounds shape_value(cv::CellValues, q_point::Int, base_func::Int) = cv.N[base_func, q_point]
 @propagate_inbounds shape_value(bv::FaceValues, q_point::Int, base_func::Int) = bv.N[base_func, q_point, bv.current_face[]]
 
+"""
+    geometric_value(fe_v::AbstractValues, q_point::Int, base_function::Int)
+
+Return the value of shape function `base_function` used for geometric interpolation evaluated in
+quadrature point `q_point`.
+"""
 @propagate_inbounds geometric_value(cv::CellValues, q_point::Int, base_func::Int) = cv.M[base_func, q_point]
 @propagate_inbounds geometric_value(bv::FaceValues, q_point::Int, base_func::Int) = bv.M[base_func, q_point, bv.current_face[]]
+@propagate_inbounds function geometric_value(iv::InterfaceValues, q_point::Int, base_func::Int) 
+    nbf_a = getngeobasefunctions(iv.face_values_a)
+    return base_func <= nbf_a ? geometric_value(iv.face_values_a, q_point, base_func) : geometric_value(iv.face_values_b, q_point, base_func - nbf_a)
+end
```

Comment:
Where is this used? Does it make sense to define it? Mapping here is quite different